web/src/components/base/Button.tsx

```diff
@@ -11,21 +11,30 @@ const StyledButton = styled.button<{ disabled: boolean }>`
 `
 
 type ButtonProps = {
-  onClick: () => void
+  onClick?: () => void
```

Comment:
AFAICS this is never undefined
```suggestion
  onClick: () => void
```

Reply:
Jep. I changed that at some poin. But now it can be reverted, I think :)